recipes/apr-util/all/conanfile.py

```diff
@@ -0,0 +1,179 @@
+            raise ConanInvalidConfiguration("CCI has no nss recipe (yet)")
+        elif self.options.crypto == "commoncrypto":
+            # self.requires("commoncrypto/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no commoncrypto recipe (yet)")
+        if self.options.dbm == "gdbm":
+            # self.requires("gdbm/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no gdbm recipe (yet)")
+        elif self.options.dbm == "ndbm":
+            # self.requires("ndbm/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no ndbm recipe (yet)")
+        elif self.options.dbm == "db":
+            # self.requires("libdb/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no libdb recipe (yet)")
+        if self.options.with_lber:
+            # self.requires("lber/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no lber recipe (yet)")
+        if self.options.with_ldap:
+            # self.requires("ldap/x.y.z")
+            raise ConanInvalidConfiguration("CCI has no ldap recipe (yet)")
+        if self.options.with_mysql:
+            self.requires("libmysqlclient/8.0.17")
+        if self.options.with_sqlite3:
+            self.requires("sqlite3/3.31.1")
+        if self.options.with_expat:
+            self.requires("expat/2.2.9")
+        if self.options.with_postgresql:
+            self.requires("libpq/11.5")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["APR_INCLUDE_DIR"] = ";".join(self.deps_cpp_info["apr"].include_paths)
+        self._cmake.definitions["INSTALL_PDB"] = False
+        self._cmake.definitions["APU_HAVE_CRYPTO"] = bool(self.options.crypto)
+        self._cmake.definitions["APR_HAS_LDAP"] = self.options.with_ldap
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        my_unix_path = tools.unix_path if tools.os_info.is_windows else lambda x: x
+        conf_args = [
+            "--with-apr={}".format(my_unix_path(self.deps_cpp_info["apr"].rootpath)),
+            "--with-crypto" if self.options.crypto else "--without-crypto",
+            "--with-expat={}".format(my_unix_path(self.deps_cpp_info["expat"].rootpath)) if self.options.with_expat else "--without-expat",
+            "--with-mysql={}".format(my_unix_path(self.deps_cpp_info["libmysqlclient"].rootpath)) if self.options.with_mysql else "--without-mysql",
+            "--with-pgsql={}".format(my_unix_path(self.deps_cpp_info["libpq"].rootpath)) if self.options.with_sqlite3 else "--without-postgresql",
```

Comment:
you have a typo hwere.
for pgsql you are checking option with_sqlite3 rather than libpq.